benchmark/citation/train_eval.py

```diff
@@ -96,10 +97,21 @@ def run_train(dataset, model, runs, epochs, lr, weight_decay, early_stopping,
         with torch_profile():
             train(model, optimizer, data)
 
+    if use_compile:
+        compiled_model = torch_geometric.compile(model)
+        benchmark(
+            funcs=[model, compiled_model],
+            func_names=['Vanilla', 'Compiled'],
+            args=data,
+            num_steps=30 if device == torch.device('cpu') else 50,
+            num_warmups=5 if device == torch.device('cpu') else 10,
+            backward=True,
+        )
```

Comment:
Not sure why we are using the `benchmark` function here. Wouldn't we first want to compile the model and then train it regularly?

Reply:
I have updated citation as kernel and points, not using `benchmark` anymore.